frontend/pnpm-workspace.yaml

```
@@ -1,6 +1,2 @@
 packages:
-  # include packages in subfolders (e.g. apps/ and packages/)
-  - "apps/**"
-  - 'packages/**'
-  # if required, exclude some directories
-  - '!**/test/**'
+  - src
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)